tests/test_hstrat/test_auxiliary_lib/test_alifestd_make_ancestor_list_col.py

```diff
@@ -8,8 +8,14 @@
     alifestd_make_ancestor_list_col,
 )
 
+from ._impl import check_polars_implementation
+
 assets_path = os.path.join(os.path.dirname(__file__), "assets")
 
+alifestd_make_ancestor_list_col = check_polars_implementation(
```

Comment:
Instead of renaming to overwrite the original, I'd rename to end with _ (alifestd_try_add_ancestor_list_col_) so there's a little hint throughout that we're using a wrapped version